tests/dummy/app/router.js

```diff
@@ -1,12 +1,14 @@
 import Ember from 'ember';
 import config from './config/environment';
 
-const Router = Ember.Router.extend({
+const { Router } = Ember;
+
+const EmRouter = Router.extend({
```

Comment:
That styleguide will soon be out of date. This is the new importing standard: https://github.com/ember-cli/ember-rfc176-data